.gitignore

```diff
@@ -1,11 +1,3 @@
 ccache/
 build/
 out/
-
```

Comment:
Please revert the changes to `.gitignore`. These are being used by the VMTests test suite.

Reply:
ah I was wondering what those are for. Missed the VM tests. 👍